dojo/engagement/views.py

```diff
@@ -1144,11 +1142,16 @@ def view_edit_risk_acceptance(request, eid, raid, edit_mode=False):
 
     unaccepted_findings = Finding.objects.filter(test__in=eng.test_set.all(), risk_accepted=False) \
         .exclude(id__in=accepted_findings).order_by("title")
-    add_fpage = get_page_items(request, unaccepted_findings, 10, 'apage')
+    add_fpage = get_page_items(request, unaccepted_findings, 500, 'apage')
```

Comment:
```suggestion
    add_fpage = get_page_items(request, unaccepted_findings, 25, 'apage')
```
500 seems excessive. Most other pages show 25 findings

Reply:
https://github.com/DefectDojo/django-DefectDojo/pull/9737/commits/1542e69fb6e34d4c6d67946e11bc358bac55e950